src/backend/expungeservice/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM library/ubuntu
+RUN apt-get update &&\
+    apt-get install -y python3.7 python3.7-dev python3-pip curl &&\
+    python3.7 -m pip install pip --upgrade &&\
+    python3.7 -m pip install wheel
+
+ #&&\
+ #curl https://raw.githubusercontent.com/kennethreitz/pipenv/master/get-pipenv.py | python3.7
+
+ADD requirements.txt /tmp
+WORKDIR /tmp
+RUN python3.7 -m pip install -r requirements.txt && rm requirements.txt
```

Comment:
Can't we invoke pipenv to install the dependencies inside docker?

Reply:
I didn't think it made sense to use pipenv inside a docker container since the container is an isolated environment, but maybe you mean why maintain two dependency files? That's a good point

Reply:
> maybe you mean why maintain two dependency files? 

Yes, I was of the idea that pipenv is supposed to replace the requirements.txt method of tracking dependencies.

It seems requirements.txt may be needed if we want to package / install the application according to https://medium.com/@greut/building-a-python-package-a-docker-image-using-pipenv-233d8793b6cc

It seems also possible to use pipenv within docker and there are some practices to follow to avoid docker rebuilding the layer repeatedly according to https://jasonfavrod.com/developing-with-docker/

I suppose we can go with what's known or simple and easy to maintain.

Reply:
I compromise and use pipenv to generate requirements.txt in the makefile, then use pip to install the generated requirements.txt in the container. This way we maintain one file of requirements. One trade-off is that we install more into the container than is probably required